aiida_lsmo/workchains/isotherm.py

```diff
@@ -2,14 +2,16 @@
 """Isotherm workchain"""
 
 import os
+import functools
+import ruamel.yaml as yaml
 
 from aiida.plugins import CalculationFactory, DataFactory, WorkflowFactory
 from aiida.orm import Dict, Str, List, SinglefileData
 from aiida.engine import calcfunction
 from aiida.engine import WorkChain, ToContext, append_, while_, if_
-from aiida_lsmo.utils import check_resize_unit_cell, aiida_dict_merge, dict_merge
+from aiida_lsmo.utils import check_resize_unit_cell, dict_merge, validate_dict
 from aiida_lsmo.utils.isotherm_molecules_schema import ISOTHERM_MOLECULES_SCHEMA
-
+from .parameters_schemas import FF_PARAMETERS_VALIDATOR, Required, Any, Optional
```

Comment:
The name `FF_PARAMETERS_VALIDATOR` is a bit confusing because it finishes containing many other settings that have not to do with the force field, and it is then easy to confuse with other dictionary that have only ff parameters. What about `SYSTEM_PARAM_VALIDATOR`?

Reply:
In which sense do you mean many settings that don't have to do with the force field?

```python
FF_PARAMETERS_VALIDATOR = voluptuous.Schema({
    Required('ff_framework',
             default='UFF',
             description='Forcefield of the structure  (used also as a definition of ff.rad for zeopp)'):
        str,
    Required('ff_separate_interactions',
             default=False,
             description='if true use only ff_framework for framework-molecule interactions in the FFBuilder'):
        bool,
    Required('ff_mixing_rule', default='Lorentz-Berthelot', description='Mixing rule'):
        Any('Lorentz-Berthelot', 'Jorgensen'),
    Required('ff_tail_corrections', default=True, description='Apply tail corrections.'):
        bool,
    Required('ff_shifted', default=False, description='Shift or truncate the potential at cutoff.'):
        bool,
    Required('ff_cutoff', default=12.0, description='CutOff truncation for the VdW interactions (Angstrom).'):
        NUMBER,
})
```

To me, these are all related to the force field (and the keys all start with `ff_`)

Reply:
Yes, but just the  ff of the system, than later has to be combined with the parameters for the molecule to make the whole force field. But ok, my proposal can also become confusing because of the allusion to (operative) system. LEt's keep 'FF_PARAMETERS_VALIDATOR'.